dateutil/test/test_relativedelta.py

```diff
@@ -314,8 +314,8 @@ def testComparison(self):
         self.assertNotEqual(d1, d3)
 
     def testInequalityTypeMismatch(self):
-        # Different type
-        self.assertFalse(relativedelta(year=1) == 19)
+        self.assertFalse(relativedelta(year=1) == None)
```

Comment:
Please add a new test instead of changing an existing one. The existing test should still be valid.

Reply:
Addressed